skaffold.yaml

```diff
@@ -22,7 +22,7 @@ build:
         dockerfile: docker/substra-tests/Dockerfile
         buildArgs:
           SUBSTRA_GIT_REPO: https://github.com/SubstraFoundation/substra.git
-          SUBSTRA_GIT_REF: master
+          SUBSTRA_GIT_COMMIT: master
```

Comment:
looks like there's a discrepancy here: the variable name is `COMMIT` but the value is `master`

Reply:
Then I should keep the 'git_ref' name in the Dockerfile since it does `pip install git+{URL}@{REF}` where REF can either be a commit or branch (but `HEAD` does not work)

Reply:
Yep that sounds good!